pyproject.toml

```diff
@@ -86,7 +86,8 @@ full = [
     "pytorch-memlab",
     "pgmpy",
     "opt_einsum",
-    "statsmodels"
+    "statsmodels",
+    "rdkit",
```

Comment:
Added this dependency to run the corresponding test case in CI. The coverage is shown less because the test case is skipped with `@withPackage('rdkit')`.